### PR TITLE
Add batch tracking sequence diagrams

### LINKED ISSUE #339: Create a sequence diagram for the DLT monitors
Create a sequence diagram for the DLT Monitors for Grid Docs, so we can document how calls are made between the various services.

### PR DESCRIPTION
**Pull request by ameliabradley**
Closes #339 

Also updates the documentation for public traits and structs

### CODE REVIEW

File: community/planning/dlt_polling_monitor.md [hunk 1]
```diff
@@ -67,6 +70,34 @@ The DLT Polling Monitor will work through the queue in the following manner:
...
+        Poller->>+DB: Update statuses
+    end
+    Gridd->>EventMonitor: Standup DLT Event Monitor
+    EventMonitor->>+Ledger: Connect to websocket DLT event stream
+    loop Every event
+        EventMonitor->>+Poller: Trigger poll
+    end
+</div>
+
 ## Public Traits and Structs
```
Comment (shannynalayna) [inline community/planning/dlt_polling_monitor.md:101]: I think this sequence diagram looks v good, but maybe we could just add a little sentence to give some clarification for what this diagram is for ? Beyond the title, that is, perhaps would just give more context for that diagram. As it's not super clear what we mean here by "other services", but just pointing out that we mean the DLTs and whatnot

Comment (ameliabradley) [reply community/planning/dlt_polling_monitor.md:104]: Added!

File: community/planning/dlt_polling_monitor.md [hunk 2]
```diff
@@ -67,6 +70,37 @@ The DLT Polling Monitor will work through the queue in the following manner:
...
 4. Reruns on the specified `poll_interval`
 
+## Interaction with other services
+
+Below is a sequence diagram that outlines how the DLT polling monitor (Poller)
+interacts with the DLT Event Monitor, Grid Daemon, Ledger, and Database.
+
+<div class="mermaid">
+sequenceDiagram
+    participant Poller as DLT Polling Monitor
```
Comment (CLEckhardt) [inline community/planning/dlt_polling_monitor.md:80]: This might be a little cleaner if Gridd came first

Comment (ameliabradley) [reply community/planning/dlt_polling_monitor.md:81]: Good point! Updated so that Gridd is the first item.

**Review by CLEckhardt — APPROVED** [review]

**Review by shannynalayna — APPROVED** [review]

**Pull request merged**